ANNOUNCEMENT.md

```diff
@@ -0,0 +1,284 @@
+susceptible to misuse when documentation is not meticulously reviewed.
+
+### Rust
+
+Rust, with its strong focus on safety, resolves these challenges at the compiler 
+level. By explicitly adding the `'publisher` lifetime in the function declaration, 
+```rust
+    fn loan(&'publisher self) -> Sample<'publisher> {}
+```
+we inform the compiler that the `Sample` can exist at most as long as the 
+`Publisher`. If a user violates this contract, attempting to destroy the 
+`Publisher` while holding a `Sample`, the compiler prevents the program from 
+compiling — providing an optimal solution without added overhead.
+
+Accidental concurrency issues, such as moving a `Sample` to another thread, are 
+also impossible in Rust. Objects intended for thread movement must implement 
+the `Send` trait, which is not implemented by default. Any attempt to violate 
+this contract is again caught by the compiler during compilation, demonstrating 
+Rust's ability to handle such problems at compile time without introducing 
+performance or implementation overhead.
+
+## Future
+
+This leads me back to the initial question: Why does the automotive domain lag 
+behind?
+
+A mere year ago, when I embarked on the development of iceoryx2, the absence of 
+a certified Rust compiler posed a critical barrier. Without this fundamental 
+component, certifying code in the automotive domain seemed nearly insurmountable. 
+Additionally, projects within the automotive sector often span years and 
+frequently involve interactions with legacy C and C++ code bases—essential 
+factors that cannot be overlooked.
+
+Today, however, the landscape has evolved significantly. We now have access to 
+a [certified Rust compiler](https://ferrous-systems.com/ferrocene/). Rust 
+facilitates seamless invocation of functions from C code bases, thanks to 
+[bindgen](https://github.com/rust-lang/rust-bindgen), and although incorporating 
+C++ code poses some challenges, it is achievable. Moreover, the reverse is also 
+true—integrating a Rust library into an existing C/C++ code base and invoking 
+[Rust functions within C](https://doc.rust-lang.org/nomicon/ffi.html) is feasible.
+
+In my estimation, the opportune moment has arrived to take the next stride and 
+emerge as pioneers in successfully certifying code for ISO 26262 (ASIL D), with 
+iceoryx2 as the ideal project for this endeavor. While this undertaking will 
+undoubtedly require time, particularly given the intricacies of inter-process 
+zero-copy communication, I approach it with confidence. Drawing on my experience 
+certifying C++ code and the insights gained from rewriting iceoryx in Rust, I 
+believe we can navigate these challenges more efficiently.
+
+Embracing Rust not only accelerates the development of new software components 
+but also reduces costs, courtesy of the modern safety and developer features it 
+offers. This, in turn, empowers our customers to write superior code swiftly, 
```

Comment:
```suggestion
offers. This, in turn, empowers our users to write superior code swiftly, 
```
Since this is an open source project `Users` might suite better